app/ai/voice/agents/breeze_buddy/agent/pipeline.py

```
@@ -92,10 +92,15 @@ async def create_services(
         Tuple of (stt_service, llm_service, tts_service)
     """
     stt_language = getattr(configurations, "stt_language", None)
+    soniox_context = getattr(configurations, "soniox_context", None)
     if stt_language:
         logger.info(f"Using STT language from template: {stt_language}")
+    if soniox_context:
+        logger.info(f"Using Soniox context from template")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)